plat/common/include/uk/plat/common/common.lds.h

```diff
@@ -129,6 +129,14 @@
 	}								\
 	uk_inittab_end = .;
 
+#define EARLYTAB_SECTION						\
```

Comment:
Ditto as in #1471. Additionally, any reason this is not named something like `BOOT_EARLYTAB_SECTION`? As in, the others seem to be defined in such a manner that just the "UK" :british_indian_ocean_territory: is missing.
```C
#define EARLYTAB_SECTION						\
	.uk_boot_earlytab :						\
	{								\
		uk_boot_earlytab_start = .;				\
		KEEP(*(SORT_BY_NAME(.uk_boot_earlytab[0-9])))		\
		uk_boot_earlytab_end = .;				\
	}
```

Reply:
I'm moving this to an library-internal linker script. I'll keep the `uk_` prefix, as it seems to be used elsewhere too:
```
 Section to Segment mapping:
  Segment Sections...
   00     .text 
   01     .rodata .uk_thread_inittab .uk_libinfo .uk_posix_clonetab .eh_frame .eh_frame_hdr .uk_ctortab .uk_inittab .uk_eventtab 
   02     .data .uk_libparam_params .posix_socket_driver_list .uk_bootinfo .bss 
   03     .tdata .tbss 
   04     .tdata 
   05     
```